lms/djangoapps/course_home_api/progress/views.py

```diff
@@ -78,26 +82,28 @@ class ProgressTabView(RetrieveAPIView):
                 learner_has_access: (bool) whether the learner has access to the subsection (could be FBE gated)
                 num_points_earned: (int) the amount of points the user has earned for the given subsection
                 num_points_possible: (int) the total amount of points possible for the given subsection
-                percent_graded: (float) the percentage of total points the user has received a grade for in a given subsection
+                percent_graded: (float) the percentage of total points the user has received a grade for in a given
+                                subsection
```

Comment:
? Just too long for the linter or something? Was it recently changed?

Reply:
Yeah line too long. I don't know why all these lints weren't caught originally? Maybe back when we stilll allowed some failures in platform?